tests/bazel_tests.bzl

```diff
@@ -219,7 +223,7 @@ _bazel_test_script = go_rule(
             ],
         ),
         "args": attr.string_list(default = []),
-        "targets": attr.string_list(mandatory = True),
+        "targets": attr.label_list(mandatory = True),
```

Comment:
Is there a way to avoid making this `label_list`? That will cause the targets to be analyzed in the workspace where `bazel test` is invoked, which isn't strictly necessary.

Reply:
Unfortunately not. We need the absolute name of the targets, which we can only get if we make this `label_list`.

Constructing the absolute name as `[Label(t) for t in targets]` in `bazel_test` does not work.